src/components/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ForComment } from '../../Types/Comment';
+import './CommentsList.scss';
+
+type Props = {
+  comments: ForComment[],
+  removeComment: (commentId: number) => void,
+};
+
+export const CommentsList: React.FC<Props> = ({
+  comments, removeComment,
+}) => {
+  return (
+    <ul className="CommentsList__list">
+      {comments && comments.map((comment: ForComment) => (
```

Comment:
```suggestion
      {comments.map((comment: ForComment) => (
```
Если массив будет пустым - то просто ничего не покажется